engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/AncientDataMotionStrategy.java

```diff
@@ -205,15 +205,15 @@ protected Answer copyObject(DataObject srcData, DataObject destData, Host destHo
                                       ", uuid: " + cacheUuid + ")");
                         cacheMgr.deleteCacheObject(srcForCopy);
                     } else {
-                        s_logger.debug("Decrease reference count of " + cacheType +
+                        if (s_logger.isTraceEnabled()) s_logger.trace("Decrease reference count of " + cacheType +
                                        " cache(id: " + cacheId + ", uuid: " + cacheUuid + ")");
                         cacheMgr.releaseCacheObject(srcForCopy);
                     }
                 }
             }
             return answer;
         } catch (Exception e) {
-            s_logger.debug("copy object failed: ", e);
+            if (s_logger.isTraceEnabled()) s_logger.trace("copy object failed: ", e);
```

Comment:
I think it's better to leave the logging level to `DEBUG`. It is easier when investigating any issues.

Reply:
resolved in upcoming commit